FAQs.md

```diff
@@ -0,0 +1,121 @@
+## FREQUENTLY ASKED QUESTIONS
+
+This is the list of Frequently asked questions for Jockey music player. It is intended to serve as a self-service utility document. This is to aid new users or already existing users of Jockey music player.
+This document may be updated from time to time to suit any new changes to the software as new developments or changes will be implemented.
+
+1. **What is Jockey Music Player?**
+
+Jockey is a music player for Android based on Google's Material Design standards. The point of Jockey is to be a simple, lightweight media player with features designed for music enthusiasts and casual listeners alike. 
+
+
+2. **What are the features of jockey music player?**
+
+Jockey music player is designed with the essential features of a music player and they include;
+
+- playlist
+- theme customization
+- equalizer
+- sleep timer
+- search tool
+
+3. **How can I navigate between songs, artists, album, genres and playlists?**
+
+You can navigate to these pages simply by swiping left or right depending on the position.
+
+4. **How do I create a create a playlist?**
+
+To create a playlist on Jockey, navigate to the playlist tab by swiping left or right. On the playlist, tab click the plus (+) sign below and two options will pop up Smart Playlist and Playlist. Click the plus sign beside playlist, input playlist name and click on create.
+
+5. **What is smart playlist?**
+
+Smart playlist as the name implies is a playlist that automatically sorts and add music to the list according to customized rules or parameters set. 
+
+6. **Can I import and export playlists on Jockey music player?**
+
+No, playlists can neither be imported nor exported.
+
+7. **How can I add songs to a playlist?**
+
+Songs can be added to a playlist from any of the tabs. Songs, Artists, Albums, Genres. To add a song, simply click the 3 dots beside the song title and select the "Add to Playlist" option and select the desired playlist you wish the song to be added to.
+
+
+8. **How do I add a song to my favorite playlist?**
+
+Jockey player doesn't have the add favorite button. To add a song to a favorite playlist, tap on the vertical dots on a song then tap on add to playlist, and tap on favorite.
```

Comment:
Seems a little odd to call out a feature that isn't implemented (same with 9 and 10 to some extent)

Reply:
I quite agree but I think point 9 should still remain though. When I first used Jockey music player and tried to highlight songs and it didn't work, i initially thought my phone had issues. I had to restart my phone and try again and when it didn't work I had to assume that the feature wasn't yet available in the app. 

for question 10, I think it's okay to let users know about it too.

I think generally it's good to tell users the whats and whats not of the app. Maybe it shouldn't be included in the app but that can be stated in the readme doc. If also you have plans to implement those features in the future it'll be good to state it too